build/Dockerfile

```diff
@@ -15,13 +15,20 @@ RUN \
  iptables \
  nodejs \
  parallel \
+ unzip \
  && \
  apt-get clean autoclean && \
  apt-get autoremove -y && \
  git clone --depth 1 https://chromium.googlesource.com/chromium/src/tools/clang && \
  clang/scripts/update.py && \
  rm -rf /tmp/*
 
+RUN \
+  cd /usr/local/bin && \
+  curl -O -J --silent https://releases.hashicorp.com/terraform/0.7.3/terraform_0.7.3_linux_amd64.zip && \
```

Comment:
It lets curl set the filename by the http header, which i guess isn't necessary in this situation eh? I've removed this but am not going to rebuild the image with the change.